modules/sdk-core/src/bitgo/utils/tss/baseTypes.ts

```diff
@@ -0,0 +1,120 @@
+import { SerializedKeyPair } from 'openpgp';
+import { IRequestTracer } from '../../../api';
+import { KeychainsTriplet } from '../../baseCoin';
+import { Keychain } from '../../keychain';
+import { Memo, WalletType } from '../../wallet/iWallet';
+
+export type TxRequestVersion = 'full' | 'lite';
+
+export interface PrebuildTransactionWithIntentOptions {
+  reqId: IRequestTracer;
+  intentType: string;
+  sequenceId?: string;
+  recipients: {
+    address: string;
+    amount: string | number;
+    tokenName?: string;
+  }[];
+  comment?: string;
+  memo?: Memo;
+  tokenName?: string;
+  nonce?: string;
+}
+
+export type TxRequestState =
+  | 'pendingApproval'
+  | 'canceled'
+  | 'rejected'
+  | 'initialized'
+  | 'pendingDelivery'
+  | 'delivered'
+  | 'pendingUserSignature'
+  | 'signed';
+
+export type TransactionState =
+  | 'initialized'
+  | 'pendingSignature'
+  | 'signed'
+  | 'held'
+  | 'delivered'
+  | 'invalidSignature'
+  | 'rejected';
+
+export type ITxRequest = {
+  txRequestId: string;
+  walletId: string;
+  walletType: WalletType;
+  version: number;
+  enterpriseId?: string;
+  state: TxRequestState;
+  date: string;
+  userId: string;
+  intent: any;
+  pendingApprovalId?: string;
+  policiesChecked: boolean;
+  signatureShares?: SignatureShareRecord[];
+  pendingTxHashes?: string[];
+  txHashes?: string[];
+  // Only available in 'lite' version
+  unsignedTxs: unknown[];
```

Comment:
why'd we change this to `unknown`?

Reply:
This would be the base type for creating TxRequest object and seems like the unsignedTx would be specific to the signature scheme being used, By using unknown, we force people to define and override with a type which makes sense for the signature scheme in hand